.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
```

Comment:
we could up to 11 here 🚀

Reply:
I was thinking that most of our CI gates were on 3.10, so kept it consistent, we can bump up to .11 or .12 when we update the majority of other gates.